src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/Utilities.cs

```diff
@@ -16,11 +16,15 @@ namespace Microsoft.DotNet.SourceBuild.SmokeTests;
 
 public static class Utilities
 {
-    public static void ExtractTarball(string tarballPath, string outputDir)
+    public static void ExtractTarball(string tarballPath, string outputDir, ITestOutputHelper outputHelper)
```

Comment:
Callers need to be updated to specify the outputHelper don't they?

Reply:
oh, yes. I had those in my dotnet/dotnet repo, but forgot to copy them back.